events/syscall/decoding_payload_in_container.go

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"os/exec"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	DecodingPayloadInContainer,
+        events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func DecodingPayloadInContainer(h events.Helper) error {
+	if h.InContainer() {
+		cmd := exec.Command("base64", "-d")

```

Comment:
What i don't like here is that the command does not exits (`base64 -d` expects the user to pass data).
We'd better use something like: `echo "test" | base64 -d` or similar!

Reply:
Yes @FedeDP , I removed that in latest commit, as it doesn't require to pass data. Let me add it again.